docs/network-flow-visibility.md

```diff
@@ -236,13 +240,18 @@ flow-aggregator.conf: |
   # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
   #flowExportInterval: 60s
   
-  # Provide the transport protocol for the flow aggregator collecting process, which is tcp or udp.
-  #aggregatorTransportProtocol: "tcp"
+  # Provide the transport protocol for the flow aggregator collecting process, which is tls, tcp or udp.
+  #aggregatorTransportProtocol: "tls"
+  
+  # Provide DNS name or IP address of flow aggregator for generating TLS certificate. It must match the flowCollectorAddr parameter in the antrea-agent config.
+  #flowAggregatorAddress: "flow-aggregator.flow-aggregator.svc"
 ```
 
-Please note that the default values for `flowExportInterval` and `aggregatorTransportProtocol`
-parameters are set to `60s` and `tcp`, respectively. Please modify them as per your
-requirements.
+Please note that the default values for `flowExportInterval`, `aggregatorTransportProtocol`, 
+and `flowAggregatorAddress` parameters are set to `60s`, `tls` and `flow-aggregator.flow-aggregator.svc`, 
+respectively. Please make sure that `aggregatorTransportProtocol` is set to `tls` and 
+`enableTLSToFlowAggregator` in `agent-agent.conf` is set to true to guarantee secure communication
+ works properly. Please modify the parameters as per your requirements. 
```

Comment:
Let' add an extra sentence after "works properly": `enableTLSToFlowAggregator` and `aggregatorTransportProtocol` must always match, so TLS must either be enabled for both sides or disabled for both sides.

Reply:
Done. Thanks